content/docs/2.9/operate/prometheus.md

```diff
@@ -6,6 +6,14 @@ weight = 200
 
 ## Prometheus Exporter Metrics
 
+### Operator 
+
+The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080` at `/metrics`. The following metrics are being gathered:
+
+- `keda_operator_trigger_totals` - Total number of triggers per trigger type.
```

Comment:
Can you add the other existing metrics please? @JorTurFer Can you elaborate on the ones available from operator framework?

Reply:
![image](https://user-images.githubusercontent.com/38671299/192446671-1d7d4fa5-2d4d-42a4-8c91-ad19d0f0ed84.png)
There are way too many of them to add here. I think this is the link that mentions them - https://book.kubebuilder.io/reference/metrics-reference.html

Maybe I can link to this in the docs?

Reply:
But as you can see, in the screenshot there are a few metrics that are exposed the go_memstat, go_info, etc. But these are not in the Kubebuilder docs, I am not sure where these are being exposed from.

Reply:
If the docs are versioned and we can show the exact metrics that we offer that's fine for me

Reply:
I don't think the docs are versioned. These are for Kubebuilder v3.

Reply:
These docs are for Kubebuilder v3, the docs for v1, v2 are hosted at a separate URL.

Reply:
Perfect!